content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -23,21 +23,26 @@ a building block.  kops builds on the kubeadm work.
 
 {{% /capture %}}
 
-{{% capture body %}}
+{{% capture prerequisites %}}
```

Comment:
nit: line 12, Possibly remove `opinionated` and add another descriptor (automated?)?
nit: line 21, could be simplified or removed. Is it important to mention `If your opinions differ ... and compare to kubeadm`?

Reply:
Note: 
There is a website for kops:
https://kops.sigs.k8s.io/